### PR TITLE
Blog app - Creating a data model✨

### PR DESCRIPTION
**Pull request by momotsow**
In this project, I used migration files to build my app schema.

These are the project requirements:

- Create a new app. Give it a meaningful name
- Initialize your project with Git
- Make sure that your project has Postgres database set up
- Build your project schema
  - `Create and run the necessary migration files`
  - `Table and column names should match the ERD diagram`
  - `Foreign keys should be included.`
  - `All columns that are foreign keys should have a corresponding index`

### CODE REVIEW

**Review by andersonlebon — COMMENTED** [review]
## Hi @momotsow 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file kindly add valid information to your README by following the microverse README template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by CollinsTatang — COMMENTED** [review]
Hi @momotsow , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have all the README file headers updated.

- [ ] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present and update its details to match your project activities.
- [ ] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [ ] Please fix the License link is broken.
- [ ] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20230223110925_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :PostsCounter
```
Comment (MahmoudBakr23) [inline db/migrate/20230223110925_create_users.rb]: - [x] Please follow the Ruby name convention to be `posts_counter` instead. Also with the `likes_counter`  and `comments_counter` in the `posts` table

_Kindly apply the changes in all similar situations_

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @momotsow 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done creating your models so far
- Nice job with adding the correct references for each table
- Descriptive PR and README file
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230223110925_create_users.rb (hunk 1 above)
Comment (momotsow) [reply db/migrate/20230223110925_create_users.rb]: @MahmoudBakr23 thank you for your review. Fixed

**Review by danmainah — APPROVED** [review]
Hi @momotsow , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**